packages/aws-cdk-lib/aws-ssm/README.md

```diff
@@ -58,6 +58,15 @@ This method uses AWS API calls to lookup the value from SSM during synthesis.
 const stringValue = ssm.StringParameter.valueFromLookup(this, '/My/Public/Parameter');
 ```
 
+The result of the `StringParameter.valueFromLookup()` operation will be written to a file
+called `cdk.context.json`. You must commit this file to source control so
+that the lookup values are available in non-privileged environments such
+as CI build steps, and to ensure your template builds are repeatable.
+To customize the cache key use the `additionalCacheKey` parameter.
+This can be useful if you want to scope the context variable to a construct 
+(eg, using `additionalCacheKey: this.node.path`).
+
+
```

Comment:
```suggestion

To customize the cache key use the `additionalCacheKey` parameter.
This can be useful if you want to scope the context variable to a construct 
(eg, using `additionalCacheKey: this.node.path`).

```ts
const stringValue = ssm.StringParameter.valueFromLookup(this, '/My/Public/Parameter', this.node.path);
```